shieldlib/README.md

```diff
@@ -62,14 +62,15 @@ The structure is as follows:
 
 ```json
 {
+    "default": { ...definiton },
```

Comment:
Typo, `definition`? Also in the next few lines.

Reply:
The idea is that it's specified in the text below

Reply:
Oh, I see the typo now.  Fix that just now.